<!DOCTYPE HTML>
<!--
	Miniport by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Shen Lab | Zhejiang University</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="pics/icon_home.png">

	</head>
	<body style="background-color:#fffcfc">

		<!-- Nav -->
		<nav id="nav">
			<ul class="container" >
			<!--
			<p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 20px"> <font face="verdana" color="#d16f2a" size="6.8">The Shen Laboratory</font></p>
			 <p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 10px"> <font face="verdana" color="#ffffff" size="5">Phylogenetics & Evolutionary
			-->
				<p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 20px"> <font face="verdana" color="#ffffff" size="6.8">The Shen Laboratory</font></p>
		    <p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 10px"> <font face="verdana" color="#ffffff" size="5">Phylogenetics & Evolutionary Genomics</font></p>
			</ul>

			<ul class="container">
			
					<li><a href="index.html" style="color: white"><font face="verdana" color="#d16f2a" size="4.8">HOME</font></a></li>
					<!--li><a href="about.html"><font face="verdana" color="#ffffff" size="4.8">ABOUT</font></a></li-->
				    <li><a href="research.html"><font face="verdana" color="#ffffff" size="4.8">RESEARCH</font></a></li>
					<li><a href="people.html"><font face="verdana" color="#ffffff" size="4.8">PEOPLE</font></a></li>
					<li><a href="publications.html"><font face="verdana" color="#ffffff" size="4.8">PUBLICATIONS</font></a></li>
					<li><a href="resources.html"><font face="verdana" color="#ffffff" size="4.8">RESOURCES</font></a></li>
					<li><a href="contact.html"><font face="verdana" color="#ffffff" size="4.8">CONTACT</font></a></li>

			</ul>
		</nav>

		<!-- Home -->

			<div class="wrapper style1 first"  style="background-color:#fffcfc">
				<article class="container" id="HOME">
					<div class="row">					    
						<p style="margin-left:1em;margin-bottom: 2px;margin-top:10px"></p>
					   
					</div>
				  <div class="row">					    
						<p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/outlliergene.jpg" alt="" height="400" width="560"></img></p>
					    
 						 <p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/proposed_evolutionary_adaptations.png" alt="" height="400" width="600"></img></p>

					</div>

                  <div class="row">

						   <p style="margin-left;line-height: 40px;font-size:19px"align="left"><strong><b><font face="verdana" size="4.5">The Shen lab is a part of the Institute of Insect Sciences at Zhejiang University. We believe that genome sequences, which are often used to understand the biodiversity that is essential to all aspects for the study of evolution and biology in general, are rich archives of history and function. Research in the lab focuses on utilizing computational approaches and genome-scale data to shed light on questions in molecular phylogenetics, evolutionary biology, comparative genomics, and bioinformatics.</font></b></strong></p> 
						
				  </div>
				  <hr style="width:100%;text-align:left;margin-left;margin-bottom: 10px;margin-top: -20px">

                 <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"> <b><font face="verdana" color="#d16f2a" size="5">Lab News</font></b></p>
				 </div>


				   <!--div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font face="verdana" color="#d16f2a" size="4.5">&#9670;6/23/2022  - Please check out new paper in <i>Cell</i> (in press)on widespread horizontal gene transfers into insects&#33;</font></b></p>
				  </div>
	
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: -60px"><img src="pics/cartoon_courtship_clean_new.png" alt="" height="450" width="550"></img>	
				   <video width="620" height="520" controls><source src="pics/Li et al. Video S1_Behavioral experiments.mp4" type="video/mp4">Your browser does not support the video tag.</video></p>
				  </div>

			      <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font size="4">In this study, we systematically examined HGT in 218 high-quality genomes of diverse insects and found that Lepidopterans had the highest average number of HGT-acquired genes (16 genes per species). HGT-acquired genes containing introns exhibited substantially higher expression levels than genes lacking introns, suggesting that intron gains were likely involved in HGT adaptation. In addition, we used the CRISPR-Cas9 system to edit the most prevalent unreported gene <i>LOC105383139</i>, which was transferred into the last common ancestor of moths and butterflies. In diamondback moths, males lacking <i>LOC105383139</i> courted females significantly less.</font></b></p>

				  </div-->
				   <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font face="verdana" color="#d16f2a" size="4.5">&#9670;8/5/2021 - Welcome Yang, Yixiao, and Guozheng&#33;</font></b></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font size="4">We are very excited that three new members, Yang Li, Yixiao Zhu, and Guozheng OU, have joined the Shen lab! Yang will focus on the evolution of foreign genes in animals. Yixiao will work on machine learning in phylogenetic inference. Guozheng will work on comparative mito-nuclear genomics across the eukaryotic tree of life.</font></b></p>

				  </div>
				 <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font face="verdana" color="#d16f2a" size="4.5">&#9670;2/24/2021 - Please check out new paper in <a target="_blank" href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syab011/6146422?searchresult=1"><i>Systematic Biology</i></a> on dissecting incongruence in phylogenomics&#33;</font></b></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/sb_workflow.png" alt="" height="450" width="400"></img></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font size="3">We developed a workflow that can a) quantify the distribution of likelihood-based signal (measured by the difference in gene-wise log likelihood score or &#916GLS) and quartet-based topological signal (measured by the difference in gene-wise quartet score or &#916GQS) for every gene in a phylogenomic data matrix, and b) identify genes that exhibit inconsistent support between concatenation- and quartet-based approaches. Our results showed that removal of inconsistent genes eliminated or extensively reduced incongruence between concatenation- and quartet-based approaches. However, it should be noted that removal of inconsistent genes in simulated datasets reduced incongruence but did not always recover the true species phylogeny.</font></b></p>

				  </div>

				 <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font face="verdana" color="#d16f2a" size="4.5">&#9670;2/19/2021 - Please check out new paper published in <a target="_blank" href="https://www.cell.com/current-biology/fulltext/S0960-9822(21)00139-1"><i>Current Biology</i></a> on fungal tree of life&#33;</font></b></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/Featured_cb_fungal_treeoflife.jpg" alt="" height="450" width="450"></img></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font size="3">In this study, we explored the genome-scale phylogeny of the fungal kingdom based on 290 genes and 1,644 species and found ~85% of inferred phylogenetic relationships among fungi are robustly supported across different subsampling matrices and tree inferences. We found several unresolved relationships (e.g., Basidiomycota) may be due to ancient diversification events. Last, we found fungal higher rank taxonomy broadly reflects organisms' genome sequence divergence, with the exception of Saccharomycotina that current taxonomic rank assignments appear to not truly reflect their evolutionary divergence.</font></b></p>
				  </div>

				 <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font face="verdana" color="#d16f2a" size="4.5">&#9670;11/30/2020 - Please check out our new phylogenomic paper published in <a target="_blank" href="https://www.nature.com/articles/s41467-020-20005-6"><i>Nature Communications</i></a> on irreproducibility in molecular phylogenetics&#33;</font></b></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/irreproducibility_signed.png" alt="" height="350" width="500"></img></p>
				  </div>

				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font size="3">In this study, we found that two replicate runs, that use exactly the same parameters (including substitution model, random starting seed number, tree search number, and log-likelihood epsilon value) on the same program (IQ-TREE or RAxML-NG), can generate substantially different maximum likelihood tree topologies. Interestingly, using 3 or more threads contributes to 2-fold increase in irreproducibility in IQ-TREE but not in RAxML-NG. Increasing reproducibility in ML inference will benefit from providing analyses' log files, which contain typically reported parameters (e.g., program, model, no. of tree searches) but also typically unreported ones (e.g., random starting seed, no. of threads, processor).</font></b></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/chinese_explain.png" alt="" height="520" width="580"></img></p>
				  </div>
				 <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font face="verdana" color="#d16f2a" size="4.5">&#9670;11/05/2020 - Our new paper just appeared in  <a target="_blank" href="https://advances.sciencemag.org/content/6/45/eabd0079"><i>Science Advances</i></a> on phylogenomics and contrasting modes of genome evolution in Ascomycota, the largest fungal phylum&#33;</font></b></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/Sci_ad.png" alt="" height="350" width="930"></img></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font size="3">In this study, we provided a comprehensive phylogeny of the Ascomycota with 1,107 publicly available genomes (Saccharomycotina: 332; Pezizomycotina: 761; Taphrinomycotina: 14) and revealed that Saccharomycotina, which contains the single currently described class Saccharomycetes, and Pezizomycotina, which contains 16 classes, exhibited greatly contrasting evolutionary modes for seven genomic properties, in particular for evolutionary rate, GC content, and genome size. Our results provide a robust evolutionary framework for understanding the fundamental genetic and ecological processes that have generated the biodiversity of the largest fungal phylum.</font></b></p>
				  </div>
			     <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font face="verdana" color="#d16f2a" size="4.5">&#9670;10/12/2020 - Xiayuhe Pei successfully applied for the graduate study at Peking University. Congrats Xiayuhe<span>&#33;</span>  </font></b></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/xpei.jpg" alt="" height="230" width="260"></img></p>
				  </div>
				  
				 <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font face="verdana" color="#d16f2a" size="4.5">&#9670;09/19/2020 - The Shen lab was funded by the NSFC grant (No. 32071665)<span>&#33;</span>  </font></b></p>
				  </div>
				  <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/NSFC_logo.jpg" alt="" height="230" width="270"></img></p>
				  </div>
				

				 <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"align="left"><b><font face="verdana" color="#d16f2a" size="4.5">&#9670;04/22/2019 - Xing-Xing Shen, 2019 Vanderbilt Postdoc of the Year Award, Honorable Mention<span>&#33;</span>  </font></b></p>
				   </div>
				 <div class="row">
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/2019 Vanderbilt Postdoc Honorable Mention.jpg" alt="" height="230" width="250"></img></p>
				   <p style="margin-left;margin-bottom: 2px;margin-top: 2px"><img src="pics/2019 Vanderbilt Postdoc Honorable Mention_2.jpg" alt="" height="230" width="250"></img></p>

				 </div>

	            <hr style="width:100%;text-align:left;margin-left;margin-bottom: -10px;margin-top: 20px">
				 <footer>
                       <p style="font-size:16px;text-align:left;;margin-bottom: -900px;"><b><font face="verdana" color="#d16f2a">&#169; All rights reserved by the Shen lab; Update December 2021</p>
                  </footer>


						
			        <!-- hitwebcounter Code START >
				    <a>
                                <img src="http://hitwebcounter.com/counter/counter.php?page=6806456&style=0006&nbdigits=6&type=ip&initCount=0" title="Visitor counter" Alt="good hits"   border="0">
                   </a-->  
				   

			 </article>
                                    
		   </div>


	</body>
</html>
