<!DOCTYPE HTML>
<!--
	Miniport by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Publications | Shen Lab</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="pics/icon_home.png">
	</head>
	<body style="background-color:#fffcfc">

		<!-- Nav -->
		<nav id="nav">
			<ul class="container" >
			<!--
			<p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 20px"> <font face="verdana" color="#d16f2a" size="6.8">The Shen Laboratory</font></p>
			-->
				<p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 20px"> <font face="verdana" color="#ffffff" size="6.8">The Shen Laboratory</font></p>
		    <p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 10px"> <font face="verdana" color="#ffffff" size="5">Phylogenetics & Evolutionary Genomics</font></p>
			</ul>

			<ul class="container">
			
					<li><a href="index.html"><font face="verdana" color="#ffffff" size="4.8">HOME</font></a></li>
					<!--li><a href="about.html"><font face="verdana" color="#ffffff" size="4.8">ABOUT</font></a></li-->
				    <li><a href="research.html"><font face="verdana" color="#ffffff" size="4.8">RESEARCH</font></a></li>
					<li><a href="people.html"><font face="verdana" color="#ffffff" size="4.8">PEOPLE</font></a></li>
					<li><a href="publications.html"><font face="verdana" color="#d16f2a" size="4.8">PUBLICATIONS</font></a></li>
					<li><a href="resources.html"><font face="verdana" color="#ffffff" size="4.8">RESOURCES</font></a></li>
					<li><a href="contact.html"><font face="verdana" color="#ffffff" size="4.8">CONTACT</font></a></li>
			</ul>
		</nav>

		
		<!-- almetric script -->
		<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>


		<!-- Publications -->
		<div class="container">
			<article id="PUBLICATIONS">
				<header>
				    <br></br>
					<br>
					<p style="margin-bottom: 2px;margin-top: 150px"><strong>Citations are available at <a target="_blank" href="https://scholar.google.com/citations?user=aNqPBsEAAAAJ&hl=en">Google Scholar</a>.</br> Lab members are indicated in <b><font face="verdana">bold</font></b>. # co-first author, * corresponding author.<strong></p>
	                <br>

                  <script>
                   var Now = new Date(),
				   Timezone = Now.getTimezoneOffset()/60, //GMT 时间与本地时间以分钟计的时间差, 换成小时； nashville：UTC-5h; China: UTC +8
				   //The time-zone offset is the difference, in minutes, between UTC and local time. Note that this means that the offset is positive if the local timezone is behind UTC and negative if it is ahead. For example, if your time zone is UTC+10 (Australian Eastern Standard Time), -600 will be returned. Daylight savings time prevents this value from being a constant even for a given locale
				   
				   CurrentDay = Now.getDay(),
                   OpeningTime = new Date(Now.getFullYear(), Now.getMonth(), Now.getDate(), 22, 00),
                   ClosingTime = new Date(Now.getFullYear(), Now.getMonth(), Now.getDate(), 7, 00),
                   Open = (Now.getTime() > OpeningTime.getTime() && Now.getTime() < ClosingTime.getTime());
				   //document.write('Today is ');
				   //document.write(Now);
				   //document.write('<br>');
				   //document.write('<br>');
				    //document.write(Timezone);
				    //document.write('<br>');
				    //document.write(CurrentDay);
				    //document.write('<br>');
				    //document.write(Now.getFullYear());
				    //document.write('<br>');
				    //document.write(Now.getMonth());
				    //document.write('<br>');
				    //document.write(Now.getDay());
				    //document.write('<br>');
				    //document.write(Now.getTime());
				    //document.write('<br>');
				    //document.write(OpeningTime.getTime());
				    //document.write('<br>');
				    // document.write(ClosingTime.getTime());
				    //document.write('<br>');
                    // days 0.sun 1.mon 2.tues 3.wed 4.thur 5.fri 6.sat 
                    // if (CurrentDay !== 6 && CurrentDay !== 0 && Open) {
					// if (Timezone == -8 && Open) {
				    //if (Timezone == -8 ) { // only display in China
                    //  document.write('<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">16. Kominek J.*, Doering D.T.*, Opulente D., <b><font face="verdana">Shen X.-X.</font></b>, Zhou X., Jeremy D., Hulfachor A.B., Mcgee M., Karlen S., Kurtzman C.P., Rokas A., Hittinger C.T. 2019. Eukaryotic acquisition of a bacterial operon. <b><font face="verdana"><i>Cell</i></font></b> in press. (*co-first author)<div style="padding-left: 1em;"></div></p>');
                   //   }
				   // </script>

				   	 <!--p style="margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">Preprint articles</font></b>
					   <hr style="width:100%;text-align:left;margin-bottom: -2px;margin-top: 20px">
					 </p>




					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      1. Walker J., <b><font face="verdana">Shen X.-X.</font></b>, Rokas A., Smith S., Moyroud E. 2020. Disentangling biological and analytical factors that give rise to outlier genes in phylogenomic matrices. <b><font face="verdana"><i>bioRxiv</i></font></b>. <a target="_blank" href="https://www.biorxiv.org/content/10.1101/2020.04.20.049999v1">bioRxiv preprint.</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1101/2020.04.20.049999" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1101/2020.04.20.049999" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p-->
	
  
					 <hr style="width:100%;text-align:left;margin-bottom: -2px;margin-top: 20px">
					 
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">2026</font></b>
					 </p>
					 
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      63. <b><font face="verdana">Chen J.#</font></b>, Feng Y.#, Zhang Y.#, Gao J.#, Ou J., <b><font face="verdana">Wu W.</font></b>, Li C., Song S., Tai L., Rifat M.H., Akhter D., Hu J., Feng P.*, <b><font face="verdana">Shen X.-X.*</font></b>, Pan R.* 2026. Structure-guided discovery of novel protein functions in plants. <b><font face="verdana"><i>The Plant Cell</i></font></b>, in press. <a target="_blank" href="pdfs/2026_Chen_etal_Plant_Cell.pdf">PDF</a><br/>
		              <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/plcell/koag022" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/plcell/koag022" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>


					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      62. <b><font face="verdana">Wu W.#</font></b>, Cui C.#, <b><font face="verdana">Zhu Y.#</font></b>, <b><font face="verdana">Chen J.</font></b>, <b><font face="verdana">Zhuang Q.</font></b>, Wang Y., Liu Z., Gao H., <b><font face="verdana">Ou G.</font></b>, <b><font face="verdana">Liu C.</font></b>, <b><font face="verdana">Tao M.</font></b>, Chen Y., Pan R., Zhang G., Cai H., Yang J., Chen X., Zhou X.*, Wang S.*, <b><font face="verdana">Shen X.-X.*</font></b> 2026. Structural genomics sheds light on protein functions and remote homologs across the insect tree of life. <b><font face="verdana"><i>Cell Research</i></font></b>, in press. <a target="_blank" href="pdfs/2026_Wu_etal_Cell_Res.pdf">PDF</a> <a target="_blank" href="https://tips.shenxlab.com/">Explore The Insect Protein Structure (TIPS) database</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s41422-026-01220-0" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s41422-026-01220-0" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 
									 
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      61. Liu Z.#, <b><font face="verdana">Tao M.#</font></b>, Xu Z.#, Zhang J.#, <b><font face="verdana">Li Y.</font></b>, Dong Z., Zhang Q., Pang L., Sheng Y., Lu Y., Feng T., Shi W., Yu L., Rokas A., Chen J.*, <b><font face="verdana">Shen X.-X.*,</font></b> Huang J.* 2026. A bacterial gene acquired by parasitoid wasps contributes to venom secretion against host defence. <b><font face="verdana"><i>The EMBO Journal</i></font></b>, in press. <a target="_blank" href="pdfs/2026_Liu_etal_The_EMBO_Journal.pdf">PDF</a><br/>

		              <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s44318-026-00702-6" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s44318-026-00702-6" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>


					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      60. Dunn C.W., Zhou X., <b><font face="verdana">Chen J.</font></b>, Musser J.M., Li Y., Haddock S.H.D., Rokhsar D.S., <b><font face="verdana">Shen X.-X.</font></b> 2026. Response: Integrative phylogenomics positions sponges at the root of the animal tree. <b><font face="verdana"><i>Science </i>(eLetter).</font></b> <a target="_blank" href="pdfs/2025_Dunn_etal_Science_eletter.pdf">PDF</a> <a target="_blank" href="https://doi.org/10.5281/zenodo.18022339">Zenodo</a> <a target="_blank" href="https://github.com/caseywdunn/sk25">GitHub</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>			 
					 
					  



					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">2025</font></b>
					 </p>		 
					 
					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      59. <b><font face="verdana">Mi X.#</font></b>, <b><font face="verdana">Ou G.#</font></b>, <b><font face="verdana">Zhu Y.</font></b>, <b><font face="verdana">Shen X.-X.*</font></b> 2025. Dissecting discordance of mitochondrial and nuclear phylogenetic trees in insects. <b><font face="verdana"><i>Crop Health</i></font></b> 3:23. <a target="_blank" href="pdfs/2025_Mi_etal_CH.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1007/s44297-025-00062-3" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1007/s44297-025-00062-3" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      58. Dong W.#, Wang S., <b><font face="verdana">Liu C.</font></b>, Qiao L., Liu K., Jin Q., Mo X., <b><font face="verdana">Shen X.-X.,</font></b> Yi K., Wang Z.* 2025. PHR-mediated Pi starvation response mobile messenger RNAs represent noncoding transcripts in recipient tissues. <b><font face="verdana"><i>Advanced Science</i></font></b> 2025: e02911. <a target="_blank" href="pdfs/2025_Dong_etal_AdvSci.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1002/advs.202502911" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1002/advs.202502911" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      57. Gonçalves C.#, Steenwyk J.L., Rinker D.C., Opulente D.A., LaBella A.L., Harrison M.-C., Wolters J.F., Zhou X., <b><font face="verdana">Shen X.-X.,</font></b> Covo S., Groenewald M., Hittinger C.T., Rokas A.* 2025. Stable hypermutators revealed by the genomic landscape of genes involved in genome stability among yeast species. <b><font face="verdana"><i>Molecular Biology and Evolution</i></font></b> 42:1-16. <a target="_blank" href="pdfs/2025_Gonçalves_etal_MBE.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/molbev/msaf285" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/molbev/msaf285" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 
					 
					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      56. Han X.#, Xu C., Ren Y., Guo M., Yan J., Wang X., <b><font face="verdana">Liu C.,</font></b> Gu Q., <b><font face="verdana">Shen X.-X.,</font></b> Ma Z., Chen Y.* 2025. Synergistic regulation by H3K36 and H3K27 methylation defines the chromatin landscape to control virulence and secondary metabolism in a fungal pathogen. <b><font face="verdana"><i>Genome Research </i></font></b> 35:2472–2487. <a target="_blank" href="pdfs/2025_Han_etal_GenomeRes.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1101/gr.280560.125" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1101/gr.280560.125" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 
					 
					 
					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      55. Wang Y.#, Song S.#, Zhang W.#, Deng Q., Feng Y., <b><font face="verdana">Tao M.,</font></b> Kang M., Zhang Q., Yang L., Wang X., Zhu C., Wang X., Zhu W., <b><font face="verdana">Zhu Y.,</font></b> Cao P., Chen J., Pan J., Feng S., Chen X., Dai H., Song S., Yang J., Zhao T., Cao F., Tao Z., <b><font face="verdana">Shen X.-X.,</font></b> Last R.L., Hu J., Yu J., Fan P.*, Pan R.* 2025. Deciphering phenylalanine-derived salicylic acid biosynthesis in plants. <b><font face="verdana"><i>Nature </i></font></b> 645(8079):208-217. <a target="_blank" href="pdfs/2025_Wang_etal_Nature.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s41586-025-09280-9" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s41586-025-09280-9" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      54. Sheng Y., Xu Z., <b><font face="verdana">Li Y.,</font></b>  Chen J., Pang L., Lu Y., Dong Z., Zhang Q., Zhang J., Feng T., Shi W., Wang Y., Chen X.*, <b><font face="verdana">Shen X.-X.*,</font></b> Huang J.* 2025. Fruit flies exploit behavioral fever as a defense strategy against parasitic insects. <b><font face="verdana"><i>Science Advances </i></font></b> 11(24):adw0191. <a target="_blank" href="pdfs/2025_Sheng_etal_Sci_adv.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1126/sciadv.adw0191" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1126/sciadv.adw0191" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>					 
			          <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      53. David K.T., Schraiber J.G., Crandall J.G., Labella A.L., Opulente D.A., Harrison M.-C., Wolters J.F., Zhou X., <b><font face="verdana">Shen X.-X.,</font></b> Groenewald M., Hittinger C.T.*, Pennell M.*, Rokas A.* 2025. Convergent expansions of keystone gene families drive metabolic innovation in Saccharomycotina yeasts. <b><font face="verdana"><i>Proc. Natl. Acad. Sci. USA</i></font></b> 122(23)e2500165122. <a target="_blank" href="pdfs/2025_Kyle_PNAS.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2500165122" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1073/pnas.2500165122" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      52. Wu Z.#, Wu X.#, Wang Z.#, Ye X., Pang L., Wang Y., Zhou Y., Chen T., Zhou S., Wang Z., Sheng Y., Zhang Q., Chen J., Tang P., <b><font face="verdana">Shen X.-X.,</font></b> Huang J., Drezen J.-M., Strand M.R., Chen X.* 2025. A symbiotic gene stimulates aggressive behavior favoring the survival of parasitized caterpillars. <b><font face="verdana"><i>Proc. Natl. Acad. Sci. USA</i></font></b> 122(18)e2422935122. <a target="_blank" href="pdfs/2025_Wu_etal_PNAS.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2422935122" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1073/pnas.2422935122" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

                      51. <b><font face="verdana">Zhu Y.,</font></b> Li Y., Li C., <b><font face="verdana">Shen X.-X.*,</font></b> Zhou X.* 2025. A critical evaluation of deep-learning based phylogenetic inference programs using simulated data sets. <b><font face="verdana"><i>Journal of Genetics and Genomics </i></font></b> 52(5):714-717. <a target="_blank" href="pdfs/2025_Zhu_JGG.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/dataset/A_Critical_Evaluation_of_Deep-Learning_Based_Phylogenetic_Inference_Programs_Using_Simulated_Data_Sets/23617767">Download data</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.jgg.2025.01.006" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1016/j.jgg.2025.01.006" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">2024</font></b>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      50. <b><font face="verdana">Liu C.#</font></b>, Zhou X.#, Li Y., Hittinger C., Pan R., Huang J.-Y., Chen X., Rokas A., Chen Y., <b><font face="verdana">Shen X.-X.*</font></b> 2024. The influence of the number of tree searches on maximum likelihood inference in phylogenomics. <b><font face="verdana"><i>Systematic Biology </i></font></b> 73(5):807-822. <a target="_blank" href="pdfs/2024_Liu_SB.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/dataset/The_influence_of_the_number_of_tree_searches_on_maximum_likelihood_inference_in_phylogenomics/17086259">Download data</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/sysbio/syae031" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/sysbio/syae031" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      49. <b><font face="verdana">Tao M.#</font></b>, Chen J.#, Cui C.#, Xu Y#, Xu J., Shi Z., Yun J., Zhang J., <b><font face="verdana">Ou G.</font></b>, <b><font face="verdana">Liu C.</font></b>, Chen Y., Zhu Z., Pan R., Xu S., Chen X., Rokas A., Zhao Y., Wang S.*, Huang J.*, <b><font face="verdana">Shen X.-X.*</font></b> 2024. Identification of a longevity gene through evolutionary rate covariation of insect mito-nuclear genomes. <b><font face="verdana"><i>Nature Aging</i></font></b> 4(8):1076–1088. <a target="_blank" href="pdfs/2024_Tao_NA.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/dataset/Identification_of_a_longevity_gene_through_evolutionary_rate_covariation_of_insect_mito-nuclear_genomes/22637761">Download data</a><br/>
	                  <div style="padding-left: 1em;">
					  <a target="_blank" href="pics/Tao_cover.png">Cover article</a>
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s43587-024-00641-z" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s43587-024-00641-z" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      48. Opulente D.A., LaBella A.L., Harrison M.-C., Wolters J.F., <font face="verdana">Liu C.</font></b>,Li Y., Kominek J., Steenwyk J.L., Stoneman H.R., VanDenAvond J., Miller C.R., Langdon Q.K., Silva M., Gonçalves C., Ubbelohde E.J., Li Y., Buh K. V., Jarzyna M., Haase M.A.B., Rosa C.A., ČCadež N., Libkind D., DeVirgilio J.H., Hulfachor A.B., Kurtzman C.P., Sampaio J.P., Gonçalves P., Zhou X., <b><font face="verdana">Shen X.-X.,</font></b> Groenewald M., Rokas A., Hittinger C.T. 2024. Genomic factors shape carbon and nitrogen metabolic niche breadth across Saccharomycotina yeasts. <b><font face="verdana"><i>Science</i></font></b> (80-. ). 384. <a target="_blank" href="pdfs/2024_Opulente_Science.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1126/science.adj4503" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1126/science.adj4503" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

			          <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      47. David K.T., Harrison M.-C., Opulente D.A., LaBella A.L., Wolters J.F., Zhou X., <b><font face="verdana">Shen X.-X.,</font></b> Groenewald M., Pennell M., Hittinger C.T., Rokas A. 2024. Saccharomycotina yeasts defy long-standing macroecological patterns. <b><font face="verdana"><i>Proc. Natl. Acad. Sci. USA</i></font></b> 121 (10) e2316031121. <a target="_blank" href="pdfs/2024_David_PNAS.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2316031121" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1073/pnas.2316031121" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      46. Harrison M.-C., Ubbelohde E.J., LaBella A.L., Opulente D.A., Wolters J.F., Zhou X., <b><font face="verdana">Shen X.-X.,</font></b> Groenewald M., Hittinger C.T., Rokas A. 2024. Machine learning enables identification of an alternative yeast galactose utilization pathway. <b><font face="verdana"><i>Proc. Natl. Acad. Sci. USA</i></font></b>  121 (18) e2315314121. <a target="_blank" href="pdfs/2024_Harrison_PNAS.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2315314121" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1073/pnas.2315314121" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

			          <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">2023</font></b>
					 </p>
					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      45. <font face="verdana">Liu C.</font></b>, Han X., Steenwyk J.L., <b><font face="verdana">Shen X.-X.*</font></b> 2023. Temporal transcriptomics provides insights into host‒pathogen interactions: a case study of <b><i>Didymella pinodella</i></font></b> and disease-resistant and disease-susceptible pea varieties. <b><font face="verdana"><i>Crop Health</i></font></b> 1:5 <a target="_blank" href="pdfs/2023_Liu_CH.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1007/s44297-023-00005-w" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1007/s44297-023-00005-w" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      44. Steenwyk J.L., Li Y., Zhou X., <font face="verdana">Shen X.-X.</font></b>, Shen X.-X., Rokas A.* 2023. Incongruence in the phylogenomics era. <b><font face="verdana"><i>Nature Reviews Genetics</i></font></b> 24:834–850 <a target="_blank" href="pdfs/2023_Steenwyk_NRG.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s41576-023-00620-x" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s41576-023-00620-x" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      43. Chen S.-J., Zhang J.-L., Ma W.-J., Wu H.-J., <font face="verdana">Li Y.</font></b>, <b><font face="verdana">Shen X.-X.</font></b>, Xu H.-J.* 2023. FoxO and rotund form a binding complex governing wing polyphenism in planthoppers. <b><font face="verdana"><i>iScience</i></font></b> 26:107182. <a target="_blank" href="pdfs/2023_Chen_iScience.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.isci.2023.107182" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1016/j.isci.2023.107182" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      42. Groenewald M.*, Hittinger C.T., Bensch K., Opulente D.A., <font face="verdana">Shen X.-X.</font></b>, Li Y., <font face="verdana">Liu C.</font></b>, LaBella A.L., Zhou X., Limtong S., Jindamorakot S., Gonçalves P., Robert V., Wolfe K.H., Rosa C.A., Boekhout T., Ĉadež N., Péter G., Sampaio J.P., Lachance M.-A., Yurkov A.M., Daniel H.-M., Takashima M., Boundy-Mills K., Libkind D., Aoki K., Sugita T., Rokas A. 2023. A genome-informed higher rank classification of the biotechnologically important fungal subphylum Saccharomycotina. <b><font face="verdana"><i>Studies in Mycology</i></font></b> 105:1-22. <a target="_blank" href="pdfs/2023_Groenewald_SIM.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.3114/sim.2023.105.01" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.3114/sim.2023.105.01" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      41. <b><font face="verdana">Liu C.</font></b>, <font face="verdana">Li Y.</font></b>, Chen Y., Chen X.-X., Huang J., Rokas A., <b><font face="verdana">Shen X.-X.*</font></b> 2023. How has horizontal gene transfer shaped the evolution of insect genomes? <b><font face="verdana"><i>Environmental Microbiology</i></font></b> 25(3):642–645. <a target="_blank" href="pdfs/2023_Liu_etal_EMB.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1111/1462-2920.16311" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1111/1462-2920.16311" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">2022</font></b>
					 </p>
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      40. Li Y.-N.*, Liu H., Steenwyk J.L., LaBella A.L., Harrison M.-C., Groenewald M., Zhou X., <b><font face="verdana">Shen X.-X.</font></b>, Zhao T., Hittinger C.T., Rokas A.* 2022. Contrasting modes of macro and microsynteny evolution in a eukaryotic subphylum. <b><font face="verdana"><i>Current Biology</i></font></b> 32(24):5335-5343. <a target="_blank" href="pdfs/2022_Li_etal_CB.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cub.2022.10.025" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1016/j.cub.2022.10.025" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      39. Steenwyk, J.L.*, Goltz, D.C., Buida, T.J., Li, Y., <b><font face="verdana">Shen X.-X.</font></b>, and Rokas, A.* 2022. OrthoSNAP: A tree splitting and pruning algorithm for retrieving single-copy orthologs from gene family trees. <b><font face="verdana"><i>PLoS biology</i></font></b> 20(10):e3001827. <a target="_blank" href="pdfs/2022_Jacob_et_al_PloS_Biol.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001827" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1371/journal.pbio.3001827" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      38. Ma, X.-Z., Wang, Z.-Q., Ye, X.-Q., Liu, X.-Y., Tang, P., <b><font face="verdana">Shen X.-X.</font></b>, and Chen, X.-X.* 2022. A high-quality genome of the dobsonfly Neoneuromus ignobilis reveals molecular convergences in aquatic insects. <b><font face="verdana"><i>Genomics</i></font></b> 114(5):110437. <a target="_blank" href="pdfs/2022_Li_etal_Cell.pdf">PDF</a><br/> 
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ygeno.2022.110437" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1016/j.ygeno.2022.110437" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      37. <b><font face="verdana">Li Y.#</font></b>, Liu Z.#, <b><font face="verdana">Liu C.#</font></b>, Shi Z., Pang L., Chen C., Chen Y., Pan R., Zhou W., Chen X.-X., Rokas A.*, Huang J.*,  <b><font face="verdana">Shen X.-X.*</font></b> 2022. HGT is widespread in insects and contributes to male courtship in lepidopterans. <b><font face="verdana"><i>Cell</i></font></b> 185(16):2975-2987. <a target="_blank" href="pdfs/2022_Li_etal_Cell.pdf">PDF</a> <a target="_blank" href="http://dx.doi.org/10.6084/m9.figshare.18094172">Download data and additional figures & tables</a>, <a target="_blank" href="https://github.com/xingxingshen/HGTfinder">HGTfinder</a><br/><a target="_blank" href="https://facultyopinions.com/article/742248834?key=8TyTflHHnvgPiIM"><img src="pics/F1000_logo.png" alt=""></a>
	                  <div style="padding-left: 1em;">
					   <a target="_blank" href="https://communities.springernature.com/posts/widespread-hgt-in-insects-a-foreign-gene-enhances-male-courtship-in-lepidopterans">The story behind the paper in Nature Ecology & Evolution blog</a><br/>
					   <b>Research Highlights in <a target="_blank" href="https://www.nature.com/articles/d41586-022-01971-x">Nature</a> and 
					   <a target="_blank" href="https://www.nature.com/articles/s41559-022-01873-1">Nature Ecology & Evolution</a></b>
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2022.06.014" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1016/j.cell.2022.06.014" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      36. Deng Q.#, Li H.#, <b><font face="verdana">Feng Y.#</font></b>, Xu R., Li W., Zhu R., Akhter D., <b><font face="verdana">Shen X.-X.</font></b>, Hu J.*, Jiang H.*, Pan R.* 2022. Defining upstream enhancing and inhibiting sequence patterns for plant peroxisome targeting signal type 1 using large‐scale <b><i>in silico</i></font></b> and <b><i>in vivo</i></font></b> analyses. <b><font face="verdana"><i>The Plant Journal</i></font></b> 111(2):567-582. <a target="_blank" href="pdfs/2022_Deng_etal_Plant_J.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1111/tpj.15840" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1111/tpj.15840" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      35. Xu S.#, Liu Y.-X.#, Cernava T.#, Wang H., Zhou Y., Xia T., Cao S., Berg G., <b><font face="verdana">Shen X.-X.</font></b>, Wen Z., Li C., Qu B., Ruan H., Chai Y., Zhou X., Ma Z., Shi Y., Yu Y.*, Bai Y.*, Chen Y.* 2022. Fusarium fruiting body microbiome member <b><i>Pantoea agglomerans</i></font></b> inhibits fungal pathogenesis by targeting lipid rafts. <b><font face="verdana"><i>Nature microbiology</i></font></b> 7(6): 831-843. <a target="_blank" href="pdfs/2022_Xu_etal_Nat_micro.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s41564-022-01131-x" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s41564-022-01131-x" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">2021</font></b>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      34. Phillips, M. A., Steenwyk J.*, <b><font face="verdana">Shen X.-X.</font></b>, Rokas A.* 2021. Examination of gene loss in the DNA mismatch repair pathway and its mutational consequences in a fungal phylum. <b><font face="verdana"><i>Genome Biology and Evolution</i></font></b> 13(10):1-12. <a target="_blank" href="pdfs/2021_Phillips_etal_GBE.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/gbe/evab219" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/gbe/evab219" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      33. Tice, A. K.#, D. Žihala#, T. Pánek#, R. E. Jones, E. Salomaki, S. Nenarokov, F. Burki, M. Eliáš, L. Eme, A. J. Roger, A. Rokas, <b><font face="verdana">Shen X.-X.</font></b>, J. F. H. Strassert, M. Kolísko,  M. W. Brown* 2021. PhyloFisher: A phylogenomic package for resolving eukaryotic relationships. <b><font face="verdana"><i>PLoS Biology</i></font></b> 19: e3001365. <a target="_blank" href="pdfs/2021_Tice_etal_PLoSBiol.pdf">PDF</a>  <a target="_blank" href="https://github.com/TheBrownLab/PhyloFisher">Download PhyloFisher</a>.<br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001365" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1371/journal.pbio.3001365" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      32. Li Y., <b><font face="verdana">Shen X.-X.</font></b>, Evans B., Dunn C.W.*, Rokas A.* 2021. Rooting the animal tree of life. <b><font face="verdana"><i>Molecular Biology and Evolution</i></font></b> 38(10):4322-4333. <a target="_blank" href="pdfs/2021_Li_etal_MBE.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/molbev/msab170" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/molbev/msab170" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       31. <b><font face="verdana">Shen X.-X.*</font></b>, Steenwyk J., Rokas A. 2021. Dissecting incongruence between concatenation- and quartet-based approaches in phylogenomic data. <b><font face="verdana"><i>Systematic Biology</i></font></b> 70(5):997-1014. <a target="_blank" href="pdfs/2021_Shen_etal_SB.pdf">PDF</a> <a target="_blank" href="https://datadryad.org/stash/dataset/doi:10.5061/dryad.9p8cz8wc5">Download data</a> <a target="_blank" href="datasets/GLS&GQS_caculation_example.zip">GLS&GQS caculation</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/sysbio/syab011" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/sysbio/syab011" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       30. Li Y., Steenwyk J., Chang Y., Wang Y., James T.Y., Stajich J.E., Spatafora J.W., Groenewald M., Dunn C., Hittinger C.T., <b><font face="verdana">Shen X.-X.*</font></b>, Rokas A.* 2021. A genome-scale phylogeny of the kingdom Fungi. <b><font face="verdana"><i>Current Biology</i></font></b> 31: 1653–1665. <a target="_blank" href="pdfs/2021_Li_etal_CB.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/dataset/Scripts_and_analyses_used_for_the_fungal_phylogeny/12751736">Download phylogenomic data</a>, <a target="_blank" href="https://doi.org/10.5281/zenodo.3970286">genome data.</a><br/>
	                  <div style="padding-left: 1em;">
					  <a target="_blank" href="pics/li_CB_cover.tif.jpg">Cover article</a>
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cub.2021.01.074" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1016/j.cub.2021.01.074" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>
			         <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      29. Steenwyk J.*, Buida III T.J., LaBella A.L., Li Y., <b><font face="verdana">Shen X.-X.</font></b>, Rokas A.* 2021. PhyKIT: a UNIX shell toolkit for processing and analyzing phylogenomic data. <b><font face="verdana"><i>Bioinformatics</i></font></b> 37(16):2325–2331. <a target="_blank" href="pdfs/2021_Steenwyk_etal_PhyKIT_Bioinf.pdf">PDF</a> <a target="_blank" href="https://jlsteenwyk.com/PhyKIT/">Download PhyKIT</a>.<br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btab096" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/bioinformatics/btab096" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">2020</font></b>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      28. <b><font face="verdana">Shen X.-X.*</font></b>, Li Y.,  Hittinger C., Chen X., Rokas A.* 2020. An investigation of irreproducibility in maximum likelihood phylogenetic inference. <b><font face="verdana"><i>Nature Communications</i></font></b> 11:6096. <a target="_blank" href="pdfs/2020_Shen_etal_Nat_Comm.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/dataset/Irreproducibility_of_maximum_likelihood_phylogenetic_inference/11917770">Download data</a>.<br/>
	                  <div style="padding-left: 1em;">
					  <a target="_blank" href="https://natureecoevocommunity.nature.com/posts/irreproducibility-in-phylogenetic-inference">The story behind the paper in Nature Ecology & Evolution blog</a>
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-020-20005-6" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s41467-020-20005-6" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      27. Steenwyk J., Buida III T.J., Li Y., <b><font face="verdana">Shen X.-X.</font></b>, Rokas A.* 2020. ClipKIT: a multiple sequence alignment-trimming algorithm for accurate phylogenomic inference. <b><font face="verdana"><i>PLoS Biology</i></font></b> 18: e3001007.  <a target="_blank" href="pdfs/2020_Steenwyk_etal_ClipKIT_PLoSBiol.pdf">PDF</a>  <a target="_blank" href="https://jlsteenwyk.com/ClipKIT/">Download ClipKT</a>.<br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001007" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1371/journal.pbio.3001007" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 </p>
					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      26. Haase M.A.B.#, Kominek J.#, Opulente D., <b><font face="verdana">Shen X.-X.</font></b>, LaBella A., Zhou X., DeVirgilio J., Hulfachor A, Kurtzman C.P., Rokas A.*, Hittinger C.T.* 2020. Repeated horizontal gene transfer of GALactose metabolism genes violates Dollo’s law of irreversible loss. <b><font face="verdana"><i>Genetics</i></font></b> 217: iyaa012.<a target="_blank" href="pdfs/2020_Haase_etal_Genetics.pdf"> PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/genetics/iyaa012" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/genetics/iyaa012" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 </p>


					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      25. Li Y.#, Davod K.T.#, <b><font face="verdana">Shen X.-X.</font></b>, Steenwyk J., Halanych K.M., Rokas A.* 2020. Feature Frequency Profile-based phylogenies are inaccurate. <b><font face="verdana"><i>Proc. Natl. Acad. Sci. USA</i></font></b> 117(50):31580-31581. <a target="_blank" href="pdfs/2020_Li_etal_PNAS.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2013143117" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1073/pnas.2013143117" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>
					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      24. <b><font face="verdana">Shen X.-X.*</font></b>, Steenwyk J., LaBella A., Opulente D., Zhou X., Kominek J., Li Y., Groenewald M., Hittinger C., Rokas A.* 2020. Genome-scale phylogeny and contrasting modes of genome evolution in the fungal phylum Ascomycota. <b><font face="verdana"><i>Science Advances </i></font></b> 6(45):eabd0079. <a target="_blank" href="pdfs/2020_Shen_etal_Sci_adv.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/dataset/Phylogenomics_and_contrasting_modes_of_genome_evolution_in_Ascomycota/12196149">Download phylogenomic data</a>, <a target="_blank" href="https://doi.org/10.5281/zenodo.3783970">genome assemblies and annotations.</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1126/sciadv.abd0079" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1126/sciadv.abd0079" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      23.  Mead M.#, Borowsky A.#, Joehnk B., Steenwyk J., <b><font face="verdana">Shen X.-X.</font></b>, Sil A., Rokas A.* 2020. Recurrent loss of abaA, a master regulator of asexual development in filamentous fungi, correlates with changes in genomic and morphological traits. <b><font face="verdana"><i>Genome Biology and Evolution</i></font></b> 12(7):1119–1130. <a target="_blank" href="pdfs/2020_Mead_etal_GBE.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/gbe/evaa107" data-hide-zero-citations="true" data-style="large_rectangle"></span>

			          <div data-badge-type="1" data-doi="10.1093/gbe/evaa107" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      22. Yang Y., Sun P., Lv L., Wang D., Ru D., Li Y., Ma T., Zhang L., <b><font face="verdana">Shen X.-X.</font></b>, Meng F., Jiao B., Shan L., Liu M., Wang Q., Qin Z., Xi Z., Wang X., Davis C.C., Liu J.* 2020. Prickly waterlily and rigid hornwort genomes shed light on early angiosperm evolution. <b><font face="verdana"><i>Nature Plants</i></font></b> 6(3):215–222. <a target="_blank" href="pdfs/2020_Yang_etal_Nat Plants.pdf">PDF</a><br/>
					  <a target="_blank" href="https://www.nature.com/articles/s41477-020-0607-5">Commentary in  <b><font face="verdana"><i>Nature Plants</i></font></b></a>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s41477-020-0594-6" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s41477-020-0594-6" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
                     <hr style="width:100%;text-align:left;margin-left:2px;margin-bottom: 2px;margin-top: 20px">
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">Before 2020 </font></b>
					 </p>
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      21. Ely Z.A., Moon J.M., Sliwoski G.R., Sangha A.K., <b><font face="verdana">Shen X.-X.</font></b>, Labella A.L., Meiler J., Capra J.A., Rokas A.* 2019. The impact of natural selection on the evolution and function of placentally expressed galectins. <b><font face="verdana"><i>Genome Biology and Evolution</i></font></b> 11(9):2574–2592 <a target="_blank" href="pdfs/2019_Ely_etal_GBE.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/gbe/evz183" data-hide-zero-citations="true" data-style="large_rectangle"></span>

			          <div data-badge-type="1" data-doi="10.1093/gbe/evz183" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      20. Krassowski, T., J. Kominek, <b><font face="verdana">Shen X.-X.</font></b>, D. A. Opulente, X. Zhou, A. Rokas, C. T. Hittinger, Wolfe K.H.* 2019. Multiple Reinventions of Mating-type Switching during Budding Yeast Evolution. <b><font face="verdana"><i>Current Biology</i></font></b> 29(15):2555–2562. <a target="_blank" href="pdfs/2019_Krassowski_etal_CB.pdf">PDF</a><br/>
					  <a target="_blank" href="https://www.sciencedirect.com/science/article/pii/S0960982219308000">Commentary in  <b><font face="verdana"><i>Current Biology</i></font></b></a>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cub.2019.06.056" data-hide-zero-citations="true" data-style="large_rectangle"></span>

			          <div data-badge-type="1" data-doi="10.1016/j.cub.2019.06.056" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

                     <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
	                 19. Wang M., Fu H., <b><font face="verdana">Shen X.-X.</font></b>, Ruan R., Rokas A., Li H.* 2019. Genomic features and evolution of the conditionally dispensable chromosome in the tangerine pathotype of <b><i>Alternaria</i></b> and <b><i>alternata</i></b>. <b><font face="verdana"><i>Mol. Plant Pathol.</i></font></b> 20(10): 1425–1438. <a target="_blank" href="pdfs/2019_Wang_etal_MPP.pdf">PDF</a>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1111/mpp.12848" data-hide-zero-citations="true" data-style="large_rectangle"></span>

			          <div data-badge-type="1" data-doi="10.1111/mpp.12848" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      18. Steenwyk, J. L., <b><font face="verdana">Shen X.-X.</font></b>, A. L. Lind, G. G. Goldman, Rokas A. 2019.*  A robust phylogenomic timetree for biotechnologically and medically important fungi in the genera <b><i>Aspergillus</i></b> and <b><i>Penicillium</i></b>. <b><font face="verdana"><i>mBio</i></font></b> 10: 1-25. <a target="_blank" href="pdfs/2019_Steenwyk_etal_mBio.pdf">PDF</a>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1128/mBio.00925-19" data-hide-zero-citations="true" data-style="large_rectangle"></span>

			          <div data-badge-type="1" data-doi="10.1128/mBio.00925-19" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      17. Steenwyk, J. L., D. A. Opulente, J. Kominek, <b><font face="verdana">Shen X.-X.</font></b>, X. Zhou, A. L. Labella, N. P. Bradley, B. F. Eichman, N. Čadež, D. Libkind, J. DeVirgilio, A. B. Hulfachor, Kurtzman C.P., Hittinger C.T., Rokas A.* 2019. Extensive loss of cell cycle and DNA repair genes in an ancient lineage of bipolar budding yeasts. <b><font face="verdana"><i>PLoS Biology</i></font></b> 17: e3000255. <a target="_blank" href="pdfs/2019_Steenwyk_etal_PLoSBiol.pdf">PDF</a>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3000255" data-hide-zero-citations="true" data-style="large_rectangle"></span>

			          <div data-badge-type="1" data-doi="10.1371/journal.pbio.3000255" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

                     <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					  16. Kominek J.#, Doering D.T.#, Opulente D., <b><font face="verdana">Shen X.-X.</font></b>, Zhou X., Jeremy D., Hulfachor A.B., Groenewald M., Mcgee M., Karlen S., Kurtzman C.P., Rokas A., Hittinger C.T.* 2019. Eukaryotic acquisition of a bacterial operon. <b><font face="verdana"><i>Cell</i></font></b> 176: 1356-1366. <a target="_blank" href="pdfs/2019_Kominek_etal_Cell.pdf">PDF</a>
                      <br/><a target="_blank" href="https://f1000.com/prime/734304746"><img src="pics/F1000_logo.png" alt=""></a><br/>
					  <a target="_blank" href="https://www.cell.com/cell/fulltext/S0092-8674(19)30156-4">Commentary in  <b><font face="verdana"><i>Cell</i></font></b></a>

	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2019.01.034" data-hide-zero-citations="true" data-style="large_rectangle"></span>

			          <div data-badge-type="1" data-doi="10.1016/j.cell.2019.01.034" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

                     <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					  15. <b><font face="verdana">Shen X.-X.#</font></b>, Opulente D.A.#, Kominek J.#, Zhou X.#, Steenwyk J., Buh K.V., Haase M., Wisecaver J.H., Wang M., Doering D.T., Boudouris J., Schneider R., Langdon Q.K., Ohkuma M., Endoh R., Takashima M., Manabe R., Čadež N., Libkind D., Rosa C., DeVirgilio J., Hulfachor A., Groenewald M., Kurtzman C.P., Hittinger C.T.*, Rokas A.* 2018. The tempo and mode of genome evolution in the budding yeast subphylum. <b><font face="verdana"><i>Cell</i></font></b> 175: 1533-1545. <a target="_blank" href="pdfs/2018_Shen_etal_Cell.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/Tempo_and_mode_of_genome_evolution_in_the_budding_yeast_subphylum/5854692">Download data</a><br/><a target="_blank" href="https://f1000.com/prime/734380463"><img src="pics/F1000_logo.png" alt=""></a><br/>
					  <a target="_blank" href="pics/Featured cell_article.png">Featured article on the homepage of the US Department of Energy Office of Science on December 4 2018 </a>
					  <br/>
					  <a target="_blank" href="https://communities.springernature.com/posts/mitigating-the-impact-of-rogue-genes-in-phylogenomic-studies">The story behind the paper in Nature Ecology & Evolution blog</a>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cell.2018.10.023" data-hide-zero-citations="true" data-style="large_rectangle"></span>

			          <div data-badge-type="1" data-doi="10.1016/j.cell.2018.10.023" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					  14. Krause D.J., Kominek J., Opulente D.A., <b><font face="verdana">Shen X.-X.</font></b>, Zhou X., Langdon Q.K., DeVirgilio J., Hulfachor A., Kurtzman C.P., Rokas A., Hittinger C.T.* 2018. Functional and evolutionary characterization of a secondary metabolite gene cluster in budding yeasts. <b><font face="verdana"><i>Proc. Natl. Acad. Sci. USA</i></font></b> 115: 11030-11035. <a target="_blank" href="pdfs/2018_Krause_etal_PNAS.pdf">PDF</a>
	                 <div style="padding-left: 1em;">
                     <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					 <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.1806268115" data-hide-zero-citations="true" data-style="large_rectangle"></span>

				     <div data-badge-type="1" data-doi="10.1073/pnas.1806268115" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
						
						
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
	                 13. Shi R., <b><font face="verdana">Shen X.-X.</font></b>, Rokas A., Eichman B.F.* 2018. Structural biology of the HEAT-like repeat family of DNA glycosylases. <b><font face="verdana"><i>BioEssays</i></font></b> 40: 1800133. <a target="_blank" href="pdfs/2018_Shi-etal_BioEssays.pdf">PDF</a>
					 <div style="padding-left: 1em;">
					 <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					 <span class="__dimensions_badge_embed__" data-doi="10.1002/bies.201800133" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					  <div data-badge-type="1" data-doi="10.1002/bies.201800133" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					</div>
					</p>
						
                     <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
	                 12. Krassowski T., Coughlan A.Y., <b><font face="verdana">Shen X.-X.</font></b>, Zhou X., Kominek J., Opulente D.A., Riley R., Grigoriev I. V., Maheshwari N., Shields D.C., Kurtzman C.P., Hittinger C.T., Rokas A., Wolfe K.H.* 2018. Evolutionary instability of CUG-Leu in the genetic code of budding yeasts. <b><font face="verdana"><i>Nature Communications</i></font></b> 9: 1887. <a target="_blank" href="pdfs/2018_Krassowski_etal_NatureComm.pdf">PDF</a>
					 <div style="padding-left: 1em;">
                     <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-018-04374-7" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					 <div data-badge-type="1" data-doi="10.1038/s41467-018-04374-7" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					</div>
					</p>
			
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					11. Gonçalves C., Wisecaver J.H., Kominek J., Oom M.S., Leandro M.J., <b><font face="verdana">Shen X.-X.</font></b>, Opulente D.A., Zhou X., Peris D., Kurtzman C.P., Hittinger C.T., Rokas A., Gonçalves P.* 2018. Evidence for loss and adaptive reacquisition of alcoholic fermentation in an early-derived fructophilic yeast lineage. <b><font face="verdana"><i>eLife</i></font></b> 7: e33034. <a target="_blank" href="pdfs/2018_Goncalves_etal_eLife.pdf">PDF</a>
					<div style="padding-left: 1em;">
					<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.7554/eLife.33034" data-hide-zero-citations="true" data-style="large_rectangle"></span>

				    <div data-badge-type="1" data-doi="10.7554/eLife.33034" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					</div>
					</p>
       
                    <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					10. Zhou X., <b><font face="verdana">Shen X.-X.</font></b>, Hittinger C.T., Rokas A.* 2018. Evaluating Fast Maximum Likelihood-Based Phylogenetic Programs Using Empirical Phylogenomic Data Sets. <b><font face="verdana"><i>Molecular Biology and Evolution</i></font></b> 35(2): 486-503. <a target="_blank" href="pdfs/2018_Zhou_mbe.pdf">PDF</a> <a target="_blank" href="https://figshare.com/projects/Evaluating_fast_maximum_likelihood-based_phylogenetic_programs_using_empirical_phylogenomic_data_sets/22040">Download data</a><br/><a target="_blank" href="https://f1000.com/prime/732181309/"><img src="pics/F1000_logo.png" alt=""></a>
					<div style="padding-left: 1em;">
					<a target="_blank" href="pdfs/2018_classic_paper.pdf">Emerging cassic article</a>
					<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.1093/molbev/msx302" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					<div data-badge-type="1" data-doi="10.1093/molbev/msx302" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					</div>
					</p>

					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					9. Shi R., Mullins E.A., <b><font face="verdana">Shen X.-X.</font></b>, Lay K.T., Yuen P.K., David S.S., Rokas A., Eichman B.F.* 2018. Selective base excision repair of DNA damage by the non-base-flipping DNA glycosylase AlkC. <b><font face="verdana"><i>EMBO Journal</i></font></b> 37: 63-74. <a target="_blank" href="pdfs/2018_Shi_etal_EMBOJ.pdf">PDF</a>
					<div style="padding-left: 1em;">
					<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.15252/embj.201797833" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					<div data-badge-type="1" data-doi="10.15252/embj.201797833" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					</div>
					</p>
		
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					8. <b><font face="verdana">Shen X.-X.</font></b>, Hittinger C. T., Rokas A.* 2017. Contentious relationships in phylogenomic studies can be driven by a handful of genes. <b><font face="verdana"><i>Nature Ecology and Evolution</i></font></b> 1: 0126. <a target="_blank" href="pdfs/2017_Shen_etal_NEE.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/Contentious_relationships_in_phylogenomic_studies_can_be_driven_by_a_handful_of_genes/3792189">Download data</a> <a target="_blank" href="datasets/Phylogenetic_signal_parser_v1.1.zip">Phylogenetic signal parser v1.1</a><br/><a target="_blank" href="https://f1000.com/prime/727814781"><img src="pics/F1000_logo.png" alt=""></a>
					<!--br/><a target="_blank" href="https://apps.webofknowledge.com/CitationReport.do?product=WOS&search_mode=CitationReport&SID=5Abk1JbLBSrArD1sRCf&page=1&cr_pqid=1&viewType=summary&colName=WOS">Most cited paper thus far at Nature Ecol & Evol reported by Web of Science</a-->
					<br/><a target="_blank" href="pdfs/Nature Ecology & Evolution's_2nd Anniversary.pdf">One of 12 evolutionary biological papers in 2017 and 2018 (Nature Ecol & Evol's 2nd Anniversary)</a>
					<br/><a target="_blank" href="pdfs/Nature Ecology & Evolution_1st Anniversary.pdf">One of 10 editors’ and readers’ favourites in 2017 (Nature Ecol & Evol's 1st Anniversary)</a><br/><a target="_blank" href="https://communities.springernature.com/posts/why-are-some-evolutionary-relationships-so-controversial">The story behind the paper in Nature Ecology & Evolution blog</a>
					<div style="padding-left: 1em;">
					<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.1038/s41559-017-0126" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					<div data-badge-type="1" data-doi="10.1038/s41559-017-0126" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>
					</div>
					</p>
	
	                <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					7. <b><font face="verdana">Shen X.-X.</font></b>, Zhou X., Kominek J., Kurtzman C. P., Hittinger C. T., Rokas A.* 2016. Reconstructing the backbone of the Saccharomycotina yeast phylogeny using genome-scale data. <b><font face="verdana"><i>G3: Genes | Genomes | Genetics</i></font></b> 6: 3927-3939. <a target="_blank" href="pdfs/2016_Shen_etal_G3.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/Reconstructing_the_backbone_of_the_Saccharomycotina_yeast_phylogeny_using_genome-scale_data/3370675">Download data</a>
					<br/><a target="_blank" href="https://academic.oup.com/g3journal/pages/10th-anniversary-collection?">One of the most cited papers in G3 (G3 10th Anniversary)</a>
					<div style="padding-left: 1em;">
					<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.1534/g3.116.034744" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					<div data-badge-type="1" data-doi="10.1534/g3.116.034744" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>
					</div>
					</p>

                    <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					6. <b><font face="verdana">Shen X.-X.</font></b>, Salichos L., Rokas A.* 2016. A genome-scale investigation of how sequence-, function-, and tree-based gene properties influence phylogenetic inference. <b><font face="verdana"><i>Genome Biology and Evolution</i></font></b> 8(8): 2565–2580. <a target="_blank" href="pdfs/2016_Shen_etal_GBE.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/A_genome-scale_investigation_of_how_sequence-_function-_and_tree-based_gene_properties_influence_phylogenetic_inference/1597710">Download data</a>
					<div style="padding-left: 1em;">
					<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.1093/gbe/evw179" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					<div data-badge-type="1" data-doi="10.1093/gbe/evw179" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>
					</div>
					</p>

                     <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					5. <b><font face="verdana">Shen X.-X.</font></b>, Liang D., Chen M.Y., Mao R.L., Wake D.B., Zhang P.* 2016. Enlarged Multilocus Dataset Provides Surprisingly Younger Time of Origin for the Plethodontidae, the Largest Family of Salamanders. <b><font face="verdana"><i>Systematic Biology</i></font></b> 65(1): 66-81. <a target="_blank" href="pdfs/2016_syst_biol.pdf">PDF</a> <a target="_blank" href="http://datadryad.org/resource/doi:10.5061/dryad.h4qn5.2/1.2">Download data</a>
					<div style="padding-left: 1em;">
	               <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.1093/sysbio/syv061" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					<div data-badge-type="1" data-doi="10.1093/sysbio/syv061" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>
					</div>
					</p>

                    <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					4. <b><font face="verdana">Shen X.-X.</font></b>, Liang D., Feng Y.J., Chen M.Y., Zhang P.* 2013. A Versatile and Highly Efficient Toolkit Including 102 Nuclear Markers for Vertebrate Phylogenomics, Tested by Resolving the Higher-level Relationships of the Caudata. <b><font face="verdana"><i>Molecular Biology and Evolution</i></font></b> 30(10): 2235-2248. <a target="_blank" href="pdfs/2013_nuclear_toolkit.pdf">PDF</a> <a target="_blank" href="datasets/2013_shen_MBE.zip">Download data</a><br/><a target="_blank" href="https://f1000.com/prime/718026842"><img src="pics/F1000_logo.png" alt="" ></a>
					<div style="padding-left: 1em;">
					<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.1093/molbev/mst122" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					<div data-badge-type="1" data-doi="10.1093/molbev/mst122" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>
					</div>
					</p>

					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					3. Liang D.#, <b><font face="verdana">Shen X.-X.#</font></b>, Zhang P.* 2013. One Thousand Two Hundred Ninety Nuclear Genes from a Genome-Wide Survey Support Lungfishes as the Sister Group of Tetrapods. <b><font face="verdana"><i>Molecular Biology and Evolution</i></font></b> 30(8): 1803-1807. <a target="_blank" href="pdfs/2013_mbe_lungfish.pdf">PDF</a> <a target="_blank" href="datasets/2013_liang_MBE.zip">Download data</a>
					<div style="padding-left: 1em;">
					<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.1093/molbev/mst072" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					<div data-badge-type="1" data-doi="10.1093/molbev/mst072" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>
					</div>
					</p>

					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					2. <b><font face="verdana">Shen X.-X.#</font></b>, Liang D.#, Zhang P.* 2012. The Development of Three Long Universal Nuclear Protein-Coding Locus Markers and Their Application to Osteichthyan Phylogenetics with Nested PCR. <b><font face="verdana"><i>PLoS ONE</i></font></b> 7(6): e39256.62. <a target="_blank" href="pdfs/2012_plos_one.pdf">PDF</a> <a target="_blank" href="datasets/2012_Shen_plosone.zip">Download data</a>
					<div style="padding-left: 1em;">
					<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pone.0039256" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					<div data-badge-type="1" data-doi="10.1371/journal.pone.0039256" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>
					</div>
					</p>

					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
				        1. <b><font face="verdana">Shen X.-X.</font></b>, Liang D, Wen J.Z., Zhang P.* 2011. Multiple Genome Alignments Facilitate Development of NPCL Markers: A Case Study of Tetrapod Phylogeny Focusing on the Position of Turtles. <b><font face="verdana"><i>Molecular Biology and Evolution</i></font></b> 28(12): 3237-3252. <a target="_blank" href="pdfs/2011_Mol.Biol.Evol.pdf">PDF</a> <a target="_blank" href="datasets/2011_shen_MBE.zip">Download data</a>
					<div style="padding-left: 1em;">
					<script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					<span class="__dimensions_badge_embed__" data-doi="10.1093/molbev/msr148" data-hide-zero-citations="true" data-style="large_rectangle"></span>

					<div data-badge-type="1" data-doi="10.1093/molbev/msr148" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>
					</div>
					</p>
				</header>
					  <hr style="width:100%;text-align:left;margin-left;margin-bottom: -10px;margin-top: 2px">
				 <footer>
                       <p style="font-size:16px;text-align:left;;margin-bottom: -900px;"><b><font face="verdana" color="#d16f2a">&#169 All rights reserved by the Shen lab</p>
                  </footer>
			</article>
		</div>





</body>
</html>
