<!DOCTYPE HTML>
<!--
	Miniport by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Publications</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="pics/icon_home.png">
	</head>
	<body>

		<!-- Nav -->
		<nav id="nav">
			<ul class="container">
					<li><a href="index.html">HOME</a></li>
					<li><a href="about.html">ABOUT ME</a></li>
					<li><a href="publications.html">PUBLICATIONS</a></li>
					<li><a href="resources.html">RESOURCES</a></li>
					<li><a href="contact.html">CONTACT</a></li>
			</ul>
		</nav>


		
		<!-- almetric script -->
		<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>


		<!-- Publications -->
		<div class="container">
			<article id="PUBLICATIONS">
				<header>
				    <br></br>
					<br></br>
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px"><strong>Publications (<a target="_blank" href="https://scholar.google.com/citations?user=aNqPBsEAAAAJ&hl=en">Google Scholar</a>)<strong></p>
					<br></br>
						
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					10. Zhou X., <u><b>Shen X.-X.,</b></u> Hittinger C.T., Rokas A. 2017. Evaluating Fast Maximum Likelihood-Based Phylogenetic Programs Using Empirical Phylogenomic Data Sets. Molecular Biology and Evolution. In press. <a href="pdfs/2017_Zhou_mbe.pdf">PDF</a> <a href="https://figshare.com/projects/Evaluating_fast_maximum_likelihood-based_phylogenetic_programs_using_empirical_phylogenomic_data_sets/22040">Download data</a><br/><a href="https://f1000.com/prime/732181309/"><img src="pics/F1000_logo.png" alt=""></a>
					<div style="padding-left: 1em;">
						<div data-badge-type="1" data-doi="10.1093/molbev/msx302" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='bottom' class="altmetric-embed"></div>	
					</div>
					</p>

					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					9. Shi R., Mullins E.A., <u><b>Shen X.-X.,</b></u> Lay K.T., Yuen P.K., David S.S., Rokas A., Eichman B.F. 2017. Selective base excision repair of DNA damage by the non-base-flipping DNA glycosylase AlkC. The EMBO Journal 1–12. <a href="pdfs/2017_Shi_etal_EMBOJ.pdf">PDF</a>
					<div style="padding-left: 1em;">
						<div data-badge-type="1" data-doi="10.15252/embj.201797833" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='bottom' class="altmetric-embed"></div>	
					</div>
					</p>
		
					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					8. <u><b>Shen X.-X.,</b></u> Hittinger C. T., Rokas A. 2017. Contentious relationships in phylogenomic studies can be driven by a handful of genes. Nature Ecology & Evolution 1: 0126. <a href="pdfs/2017_Shen_etal_NEE.pdf">PDF</a> <a href="https://figshare.com/articles/Contentious_relationships_in_phylogenomic_studies_can_be_driven_by_a_handful_of_genes/3792189">Download data</a><br/><a href="https://f1000.com/prime/727814781"><img src="pics/F1000_logo.png" alt=""></a><br/><a href="https://natureecoevocommunity.nature.com/users/25228-antonis-rokas/posts/16057-why-are-some-evolutionary-relationships-so-controversial">The story behind the paper</a>
					<div style="padding-left: 1em;">
					<div data-badge-type="1" data-doi="10.1038/s41559-017-0126" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='bottom' class="altmetric-embed"></div>
					</div>
					</p>
	
	                                <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					7. <u><b>Shen X.-X.,</b></u> Zhou X., Kominek J., Kurtzman C. P., Hittinger C. T., Rokas A. 2016. Reconstructing the backbone of the Saccharomycotina yeast phylogeny using genome-scale data. G3: Genes | Genomes | Genetics 6: 3927-3939. <a href="pdfs/2016_Shen_etal_G3.pdf">PDF</a> <a href="https://figshare.com/articles/Reconstructing_the_backbone_of_the_Saccharomycotina_yeast_phylogeny_using_genome-scale_data/3370675">Download data</a>
					<div style="padding-left: 1em;">
						<div data-badge-type="1" data-doi="10.1534/g3.116.034744" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='bottom' class="altmetric-embed"></div>
					</div>
					</p>

                                        <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					6. <u><b>Shen X.-X.,</b></u> Salichos L., Rokas A. 2016. A genome-scale investigation of how sequence-, function-, and tree-based gene properties influence phylogenetic inference. Genome Biology and Evolution 8(8): 2565–2580. <a href="pdfs/2016_Shen_etal_GBE.pdf">PDF</a> <a href="https://figshare.com/articles/A_genome-scale_investigation_of_how_sequence-_function-_and_tree-based_gene_properties_influence_phylogenetic_inference/1597710">Download data</a>
					<div style="padding-left: 1em;">
						<div data-badge-type="1" data-doi="10.1093/gbe/evw179" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='bottom' class="altmetric-embed"></div>
					</div>
					</p>

                                        <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					5. <u><b>Shen X.-X.,</b></u> Liang D., Chen M.Y., Mao R.L., Wake D.B., Zhang P. 2016. Enlarged Multilocus Dataset Provides Surprisingly Younger Time of Origin for the Plethodontidae, the Largest Family of Salamanders. Systematic Biology 65(1): 66-81. <a href="pdfs/2016_syst_biol.pdf">PDF</a> <a href="http://datadryad.org/resource/doi:10.5061/dryad.h4qn5.2/1.2">Download data</a>
					<div style="padding-left: 1em;">
					<div data-badge-type="1" data-doi="10.1093/sysbio/syv061" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='bottom' class="altmetric-embed"></div>
					</div>
					</p>

                                       <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					4. <u><b>Shen X.-X.,</b></u> Liang D., Feng Y.J., Chen M.Y., Zhang P. 2013. A Versatile and Highly Efficient Toolkit Including 102 Nuclear Markers for Vertebrate Phylogenomics, Tested by Resolving the Higher-level Relationships of the Caudata. Molecular Biology and Evolution 30(10): 2235-2248. <a href="pdfs/2013_nuclear_toolkit.pdf">PDF</a> <a href="datasets/2013_shen_MBE.zip">Download data</a><br/><a href="https://f1000.com/prime/718026842"><img src="pics/F1000_logo.png" alt="" ></a>
					<div style="padding-left: 1em;">
						<div data-badge-type="1" data-doi="10.1093/molbev/mst122" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='bottom' class="altmetric-embed"></div>
					</div>
					</p>

					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					3. Liang D.*, <u><b>Shen X.-X.*,</b></u> Zhang P. 2013. One Thousand Two Hundred Ninety Nuclear Genes from a Genome-Wide Survey Support Lungfishes as the Sister Group of Tetrapods. Molecular Biology and Evolution 30(8): 1803-1807. (* equal contribution) <a href="pdfs/2013_mbe_lungfish.pdf">PDF</a> <a href="datasets/2013_liang_MBE.zip">Download data</a>
					<div style="padding-left: 1em;">
						<div data-badge-type="1" data-doi="10.1093/molbev/mst072" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='bottom' class="altmetric-embed"></div>
					</div>
					</p>

					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
					2. <u><b>Shen X.-X.*,</b></u> Liang D.*, Zhang P. 2012. The Development of Three Long Universal Nuclear Protein-Coding Locus Markers and Their Application to Osteichthyan Phylogenetics with Nested PCR. PLoS ONE 7(6): e39256.62. (* equal contribution) <a href="pdfs/2012_plos_one.pdf">PDF</a> <a href="datasets/2012_Shen_plosone.zip">Download data</a>
					<div style="padding-left: 1em;">
						<div data-badge-type="1" data-doi="10.1371/journal.pone.0039256" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='bottom' class="altmetric-embed"></div>
					</div>
					</p>

					<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
				        1. <u><b>Shen X.-X.,</b></u> Liang D, Wen J.Z., Zhang P. 2011. Multiple Genome Alignments Facilitate Development of NPCL Markers: A Case Study of Tetrapod Phylogeny Focusing on the Position of Turtles. Molecular Biology and Evolution 28(12): 3237-3252. <a href="pdfs/2011_Mol.Biol.Evol.pdf">PDF</a> <a href="datasets/2011_shen_MBE.zip">Download data</a>
					<div style="padding-left: 1em;">
						<div data-badge-type="1" data-doi="10.1093/molbev/msr148" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='bottom' class="altmetric-embed"></div>
					</div>
					</p>
				</header>
			</article>
		</div>





</body>
</html>
