<!DOCTYPE HTML>
<!--
	Miniport by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Research | Shen Lab</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="pics/icon_home.png">
	</head>
	<body style="background-color:#fffcfc">

		<!-- Nav -->
		<nav id="nav">
			<ul class="container" >
			<!--
			<p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 20px"> <font face="verdana" color="#d16f2a" size="6.8">The Shen Laboratory</font></p>
			-->
				<p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 20px"> <font face="verdana" color="#ffffff" size="6.8">The Shen Laboratory</font></p>
		    <p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 10px"> <font face="verdana" color="#ffffff" size="5">Phylogenetics & Evolutionary Genomics</font></p>
			</ul>

			<ul class="container">
			
					<li><a href="index.html"><font face="verdana" color="#ffffff" size="4.8">HOME</font></a></li>
					<!--li><a href="about.html"><font face="verdana" color="#ffffff" size="4.8">ABOUT</font></a></li-->
				    <li><a href="research.html"><font face="verdana" color="#d16f2a" size="4.8">RESEARCH</font></a></li>
					<li><a href="people.html"><font face="verdana" color="#ffffff" size="4.8">PEOPLE</font></a></li>
					<li><a href="publications.html"><font face="verdana" color="#ffffff" size="4.8">PUBLICATIONS</font></a></li>
					<li><a href="resources.html"><font face="verdana" color="#ffffff" size="4.8">RESOURCES</font></a></li>
					<li><a href="contact.html"><font face="verdana" color="#ffffff" size="4.8">CONTACT</font></a></li>
			</ul>
		</nav>

		<!-- Publications -->
		<div class="container" style="background-color:#fffcfc">
			<article id="RESEARCH">
				    <br><br><br>
					<p style="margin-top: 120px" align="left"><strong><font color="#d16f2a" size="5.5">1. Dissecting factors that contribute to the phylogenetic uncertainties</font><strong></p>
					<div class="row">

					<div class="3u 12u(mobile)">
					<p><img src="pics/shen2017_2.png" alt="" height="240" width="300"></img></p>
					<p><img src="pics/irreproducibility_signed.png" alt="" height="240" width="300"></img></p>
					</div>
					<div class="7u 12u(mobile)">
					<p style="line-height: 30px;font-size:17px" align="left"><strong>Phylogenetic trees are essential for understanding how genes, genomes, organisms, and species evolve, yet their reliability and robustness still receive challenges across different methods and studies, even when genome-scale data are adopted. After tackling these challenges, we found the causes of uncertainties are: <br>1) A tiny subsets of very large data matrices can exhibit a disproportionate amount of influence and drive the resolution of contentious branches through quantifying their distribution of phylogenetic signal among genes and sites from phylogenomic data (<a target="_blank" href="https://shenxlab.com/publications.html">Shen et al. 2017, Nat. Ecol. Evol.</a>).<br><br>2) Low phylogenetic informativeness (e.g., low percentage of parsimony-informative sites and low branch support values), processor type, and multithreading can contribute to a considerable fraction of irreproducible single-gene ML trees, even when two replicates ran with exactly the same parameters including sequence alignment, program, substitution model, random starting seed number, number of tree search, and log-likelihood epsilon value. (<a target="_blank" href="https://shenxlab.com/publications.html">Shen et al. 2020, Nature Comm.</a>).<strong></p>
				    </div>

		

					<p style="margin-top: 10px" align="left"><strong><font color="#d16f2a" size="5.5">2. Inferring species phylogeny with genome-scale data</font><strong></p>
					<div class="row">

					<div class="3u 12u(mobile)">
					<p><img src="pics/y1000_fasttree_2_small.png" alt="" height="270" width="270"></img></p>
					</div>
					<div class="7u 12u(mobile)">
					<p style="line-height: 30px;font-size:17px" align="left"><strong>With the advances of genomic tools and affordable high throughput sequencing, phylogenomic analyses based on data matrices comprised of hundreds to thousands of genes from dozens of taxa have witnessed brisk progresses in resolving the tree of life such as placement of lungfishes (<a target="_blank" href="https://shenxlab.com/publications.html">Liang et al. 2013, Mol. Biol. Evol.</a>), phylogenomics of Ascomycota (<a target="_blank" href="https://shenxlab.com/publications.html">Shen et al. 2020, Science Advances</a>). More recently, one of my projects is involved in the Y1000+ Project (<a href="https://y1000plus.wei.wisc.edu/">http://y1000plus.org</a>) that focuses on sequencing the genomes of the entire subphylum Saccharomycotina of yeasts (>1,000) and understanding the evolution of their biodiversity (<a href="https://shenxlab.com/publications.html">Shen et al. 2018, Cell</a>).<strong></p>
				    </div>
				  </div>
				  	<!--br>
				  	<p style=align="left"><strong><font size="5.5">3. Universal nuclear marker toolkit for vertebrate phylogenomics</font><strong></p>
					<div class="row">

					<div class="3u 12u(mobile)">
					<p><img  src="pics/shen2013.png" alt="" height="370" width="260"></img></p>
					</div>
					<div class="9u 12u(mobile)">
					<p style="line-height: 30px;font-size:17px" align="left"><strong>My Ph.D. dissertation research at Sun Yat-sen University focused on developing universal nuclear markers for the vertebrate tree of life. At that time, for vertebrate phylogenetics, the most widely used approach for building multi-locus data sets is still conventional targeted polymerase chain reaction (PCR) and the sequencing of selected orthologous genes. However, the PCR-based method is laborious due to primer design, PCR optimization, gel-purification, and cloning. On the basis of multiple genome alignments (<a href="https://shenxlab.com/publications.html">Shen et al. 2011, Mol. Biol. Evol.</a>) and an efficient Nested PCR strategy (<a href="https://shenxlab.com/publications.html">Shen et al. 2012, Plos One</a>), we designed a toolkit which contains 102 universal nuclear genes for the vertebrate phylogenomics (<a href="https://shenxlab.com/publications.html">Shen et al. 2013, Mol. Biol. Evol.</a>). This toolkit has provided relatively high utilities (including amplification efficiency and phylogenetic resolution) in salamanders (<a href="https://shenxlab.com/publications.html">Shen et al. 2016, Syst. Biol.</a>) and frogs (<a target="_blank" href="http://www.pnas.org/content/114/29/E5864.abstract">Feng et al. 2017, PNAS</a>).<strong></p>
				    </div>
				  </div>
				  <br-->


			</article>
								  <hr style="width:100%;text-align:left;margin-left;margin-bottom: -10px;margin-top:2px">
				 <footer>
                       <p style="font-size:16px;text-align:left;;margin-bottom: -1200px;"><b><font face="verdana" color="#d16f2a">&#169 All rights reserved by the Shen lab</p>
                  </footer>
		</div>




</body>
</html>
