<!DOCTYPE HTML>
<!--
	Miniport by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Research | Shen Lab</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="pics/icon_home.png">
	</head>
	<body>

		<!-- Nav -->
		<nav id="nav">
			<ul class="container">
			
				<p style="text-indent:-1em;margin-left;margin-bottom: 2px;margin-top: 20px"> <font face="verdana" color="#ba7849" size="7">The Shen Laboratory</font></p>
		    <p style="text-indent:-1em;margin-left;margin-bottom: 2px;margin-top: 10px"> <font face="verdana" color="#ba7849" size="5">Phylogenetics & Evolutionary Genomics</font></p>
			</ul>

			<ul class="container">
			
					<li><a href="index.html"><font face="verdana" color="#545252" size="4.8">HOME</font></a></li>
					<!--li><a href="about.html"><font face="verdana" color="#545252" size="4.8">ABOUT</font></a></li-->
				    <li><a href="research.html"><font face="verdana" color="#e3dcdc" size="4.8">RESEARCH</font></a></li>
					<li><a href="people.html"><font face="verdana" color="#545252" size="4.8">PEOPLE</font></a></li>
					<li><a href="publications.html"><font face="verdana" color="#545252" size="4.8">PUBLICATIONS</font></a></li>
					<li><a href="resources.html"><font face="verdana" color="#545252" size="4.8">RESOURCES</font></a></li>
					<li><a href="contact.html"><font face="verdana" color="#545252" size="4.8">CONTACT</font></a></li>
			</ul>
		</nav>

		<!-- Publications -->
		<div class="container">
			<article id="RESEARCH">
				    <br><br><br>
					<p style="margin-top: 120px" align="left"><strong><font size="5.5">1. Dissecting factors that contribute to the phylogenetic incongruence</font><strong></p>
					<div class="row">

					<div class="3u 12u(mobile)">
					<p><img src="pics/shen2017.png" alt="" height="240" width="300"></img></p>
					</div>
					<div class="9u 12u(mobile)">
					<p style="line-height: 30px;font-size:17px" align="left"><strong>Phylogenomic studies have resolved countless branches of the tree of life, but remain strongly contradictory on certain, contentious relationships. Here, we use a maximum likelihood framework to quantify the distribution of phylogenetic signal among genes and sites for 17 contentious branches in plant, animal and fungal phylogenomic data matrices (<a href="https://xingxingshen.github.io/publications.html">Shen et al. 2017, Nat. Ecol. Evol.</a>). We find that resolution in some of these 17 branches tiny subsets of very large data matrices can drive the resolution of specific internodes. Therefore, quantifying and visualizing the distribution of phylogenetic signal would also be helpful for the identification of any sites or genes that might be exerting a disproportionate amount of influence on the resolution of a given contentious branch.<strong></p>
				    </div>
				  </div>

		

					<p style="margin-top: 10px" align="left"><strong><font size="5.5">2. Inferring species phylogeny with genome-scale data</font><strong></p>
					<div class="row">

					<div class="3u 12u(mobile)">
					<p><img src="pics/y1000_fasttree.png" alt="" height="240" width="240"></img></p>
					</div>
					<div class="9u 12u(mobile)">
					<p style="line-height: 30px;font-size:17px" align="left"><strong>With the advances of genomic tools and affordable high throughput sequencing, phylogenomic analyses based on data matrices comprised of hundreds to thousands of genes from dozens of taxa have witnessed brisk progresses in resolving the tree of life such as placement of lungfishes (<a href="https://xingxingshen.github.io/publications.html">Liang et al. 2013, Mol. Biol. Evol.</a>), the backbone of 86-yeast phylogeny (<a href="https://xingxingshen.github.io/publications.html">Shen et al. 2016, G3</a>). More recently, one of my projects is involved in the Y1000+ Project (<a href="https://y1000plus.wei.wisc.edu/">http://y1000plus.org</a>) that focuses on sequencing the genomes of the entire subphylum Saccharomycotina of yeasts (>1,000) and understanding the evolution of their biodiversity (<a href="https://xingxingshen.github.io/publications.html">Shen et al. 2018, Cell</a>).<strong></p>
				    </div>
				  </div>
				  	<!--br>
				  	<p style=align="left"><strong><font size="5.5">3. Universal nuclear marker toolkit for vertebrate phylogenomics</font><strong></p>
					<div class="row">

					<div class="3u 12u(mobile)">
					<p><img  src="pics/shen2013.png" alt="" height="370" width="260"></img></p>
					</div>
					<div class="9u 12u(mobile)">
					<p style="line-height: 30px;font-size:17px" align="left"><strong>My Ph.D. dissertation research at Sun Yat-sen University focused on developing universal nuclear markers for the vertebrate tree of life. At that time, for vertebrate phylogenetics, the most widely used approach for building multi-locus data sets is still conventional targeted polymerase chain reaction (PCR) and the sequencing of selected orthologous genes. However, the PCR-based method is laborious due to primer design, PCR optimization, gel-purification, and cloning. On the basis of multiple genome alignments (<a href="https://xingxingshen.github.io/publications.html">Shen et al. 2011, Mol. Biol. Evol.</a>) and an efficient Nested PCR strategy (<a href="https://xingxingshen.github.io/publications.html">Shen et al. 2012, Plos One</a>), we designed a toolkit which contains 102 universal nuclear genes for the vertebrate phylogenomics (<a href="https://xingxingshen.github.io/publications.html">Shen et al. 2013, Mol. Biol. Evol.</a>). This toolkit has provided relatively high utilities (including amplification efficiency and phylogenetic resolution) in salamanders (<a href="https://xingxingshen.github.io/publications.html">Shen et al. 2016, Syst. Biol.</a>) and frogs (<a target="_blank" href="http://www.pnas.org/content/114/29/E5864.abstract">Feng et al. 2017, PNAS</a>).<strong></p>
				    </div>
				  </div>
				  <br-->
			</article>
		</div>




</body>
</html>
